sotodlib/mapmaking/ml_mapmaker.py

```diff
@@ -161,7 +170,8 @@ def __init__(self, name, ofmt, output, ext):
         self.ext    = ext
         self.dof    = None
         self.ready  = False
-    def add_obs(self, id, obs, nmat, Nd): pass
+        self.glitch_flags = glitch_flags
+    def add_obs(self, id, obs, nmat, Nd, glitch_flags:Optional[str]): pass
```

Comment:
Same question.

Can it be replaced with a generic **kwargs, to imply "yes sometimes a subclass will accept and process other random args."

Reply:
Yes, I can